.github/scripts/ext-debian.sh

```diff
@@ -68,108 +98,123 @@ build_bin_package() {
   
   pushd .
   echo "Building binary package $PACKAGE_NAME"
-  cd "$PKG_BUILD_DIR/$PACKAGE_NAME" || exit
+  cd "$PKG_BUILD_PATH/$PACKAGE_NAME" || exit
 
   debuild -sa -b
   popd
 }
 
-# Internal (private) function
-source_pkg_exists() {
-    SRC_PKG_VERSION=$(reprepro --basedir "$PKG_REPO_PATH" list "$CODENAME" "$1" | cut -d' ' -f3)
-
-    SRC_PKG_BUILD_VERSION=$(echo $2 | cut -d'-' -f1)
-    SRC_PKG_REPO_VERSION=$(echo $SRC_PKG_VERSION | cut -d'-' -f1)
-
-    if [ "$SRC_PKG_REPO_VERSION" == "$SRC_PKG_BUILD_VERSION" ]; then
-        return 0
-    else
-        return 1
-    fi
-}
-
 publish() {
-  cd "${PKG_BUILD_DIR:?}/$PACKAGE_NAME"
+  cd "${PKG_BUILD_PATH:?}/$PACKAGE_NAME"
   version=$(dpkg-parsechangelog --show-field Version)
   debian_package_name=$(dpkg-parsechangelog --show-field Source)
-  cd "$PKG_BUILD_DIR"
+  cd "$PKG_BUILD_PATH"
 
-  DEB_SRC_PKG_PATH="$PKG_BUILD_DIR/${debian_package_name}_${version}_source.changes"
+  DEB_SRC_PKG_PATH="$(pwd)/${debian_package_name}_${version}_source.changes"
 
   if [ ! -f "$DEB_SRC_PKG_PATH" ]; then
     echo "Failed to find changes file."
-  fi
-
-  if source_pkg_exists "$debian_package_name" "$version"; then
-      echo "Ignoring source package, already exists in target repository"
-      allow_failing_bin_package="true"
   else
-      echo "Ingesting source package $debian_package_name into $PKG_REPO_PATH"
-      reprepro --basedir "$PKG_REPO_PATH" include "$CODENAME" "$DEB_SRC_PKG_PATH"
-      allow_failing_bin_package="false"
+    short_version="${version%-*}"
+
+    mkdir -p $PKG_PUBLISH_PATH/$DISTRO/$CODENAME/$SUITE
```

Comment:
question: can you explain what `SUITE` models?  How is it different than stage?

Reply:
`SUITE` is introduced to be able to support multiple Regolith version per release `STAGE`. Suite is, very loosely, the same as Stage with one big difference. Stage is release stage from the SDLC point of view, but Suite is from the point of view of the published debian package archives.

The corresponding values and mappings are as follow:

| Stage | Suite | Component |
|----------|---------|------------------|
| `experimental` | `experimental` | `main` |
| `unstable` | `unstable` | `main` |
| `testing` | `testing` | `main` |
| `release-current` | `stable` | `main` |
| `release-3_2` | `stable` | `v3.2` |
| `release-3_1` | `stable` | `v3.1` |

Or in other words, in the scope of deb config line:

```text
deb [arch=amd64] http://archive.regolith-desktop.com/ubuntu/unstable noble main
          |                                          |      |        |     |
          |                                          |      |        |     * Component
          |                                          |      |        * Distro Codename
          |                                          |      * Regolith Suite
          |                                          * Distro Name
          * Architecture
```

Reply:
LMK if this is not the case: `SUITE` is a debian repo concept.  We would not necessarily use or need it for other packaging systems.